src/appkit.cs

```diff
@@ -3891,6 +3973,21 @@ public interface NSColorSpace : NSCoding, NSSecureCoding {
 		[Export ("availableColorSpacesWithModel:")]
 		NSColorSpace [] AvailableColorSpacesWithModel (NSColorSpaceModel model);
 
+		[Mac (10, 12)]
+		[Static]
+		[Export ("extendedSRGBColorSpace")]
+		NSColorSpace ExtendedSRGBColorSpace { get; }
```

Comment:
we already have `NSColor.FromSrgb` and other cases of `Srgb` elsewhere (e.g. coregraphics and metalkit) and no `SRgb` so it should be `ExtendedSrgbColorSpace` imo